docs/building_pybindings.txt

```diff
@@ -92,3 +92,14 @@ Run tests:
   MWM_RESOURCES_DIR=data \
   MWM_WRITABLE_DIR=data \
   search/pysearch/run_search_engine.py
+
+Problems and solutions
+----------------------
+1. Building binding on MacOS Catalina 10.15.5.
+   If you try to build one of the bindings on MacOS Catalina with Apple clang version 11.0.3 you'll see the error that the flag fcoalesce-templates is not supported. The workaround is to remove this flag from 3party/boost/tools/build/src/tools/darwin.jam and 3party/boost/tools/build/src/tools/darwin.py
+
+2. If there is a build errors while Boost.Python building the C++ headers are not seen. For example,  fatal error: 'cstddef' file not found. Some flags should be added while building bindings. For example, instead launching:
```

Comment:
``there is a build error`` или ``there are build errors``.
И структуру первого предложения вообще не понял.

``Some flags should be added`` слишком неконкретно, кмк. ``You have to add following flags:``, что-то такое.

``instead launching`` -> ``instead of launching``

Reply:
`2. You may get build errors that standard C++ headers are not seen while
   building of Boost.Python. For example, fatal error: 'cstddef' file not found.
   In that case the following flags should be set to correct values:
   MACOSX_DEPLOYMENT_TARGET, CC, CXX, CFLAGS. For example, instead of launching:`